packages/xo-web/package.json

```diff
@@ -27,7 +27,7 @@
     ">2%"
   ],
   "engines": {
-    "node": ">=6"
+    "node": ">=7"
```

Comment:
Why this change? :thinking:

Reply:
because of a linter warning on missing object.entries in one of xo-web files
It should have been another PR, but I really don't like to get used to commit with no-verifiy

Reply:
It's not really relevant for this package, but it makes sense, thanks :slightly_smiling_face: